book/tutorials/Data_access/earthaccess_snowex.ipynb

```diff
@@ -0,0 +1,291 @@
+    "| SNEX23_SWE          | SnowEx23 Snow Water Equivalent |\n",
+    "| SNEX23_MAR23_SD     | SnowEx23 Mar23 IOP Community Snow Depth Measurements |\n",
+    "| SNEX23_SSA          | SnowEx23 Laser Snow Microstructure Specific Surface Area Data |\n",
+    "| SNEX23_UW_GPR       | SnowEx23 University of Wyoming Ground Penetrating Radar"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Narrow Search using a Spatial Filter\n",
+    "\n",
+    "Here we are going to use a bounding box for the Fairbanks study area to find SnowEx collections."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Query = earthaccess.search_datasets( \n",
+    "  keyword = 'SnowEx',\n",
+    "     bounding_box = (-149.597,64.699,-147.49,70.085),\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Eleven datasets were found within this bounding box.<br> As we did above, we will make a list of the datasets within this bounding box."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "for collection in Query[:11]:\n",
+    "    pprint.pprint(collection.summary(), sort_dicts=True, indent=4)\n",
+    "    print('')\n",
+    "    "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 3:  Search for data granules (files)\n",
+    "First we will search for the SnowEx 23 Mar22 IOP Snow Depth Measurement collection using its short name: Snex23_Mar22_SD.   "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "results = earthaccess.search_data(\n",
+    "    short_name='Snex23_MAR22_SD',\n",
+    "    bounding_box=(-149.597,64.699,-147.49,70.085)\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "results"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 4: Download Data Locally\n",
+    "For this section, we will download the granule above from the SnowEx23 Mar22 IOP Snow Depth Measurements collection locally. <br>\n",
+    "\n",
+    "We'll download the file into a separate folder named \"Downloads\", which will be created for us, if it doesn't already exist."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally, we will go ahead and download the granule locally to this notebook into a separate folder named \"Downloads.\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "downloaded_files = earthaccess.download(\n",
+    "    results,\n",
+    "    local_path='./Downloads',\n",
```

Comment:
Consider changing the download path to `/tmp` for tutorial data (https://docs.2i2c.org/user/topics/data/filesystem/#the-tmp-directory)